release.sh

```diff
@@ -8,35 +8,36 @@ YELLOW='\033[0;33m'
 CYAN='\033[0;36m'
 NC='\033[0m' # No color
 
-echo "${GREEN}Bumping version...${NC}"
-yarn lerna version \
-  --no-push \
-  --no-git-tag-version \
-  --force-publish=@cmsgov/design-system,@cmsgov/design-system-docs,@cmsgov/design-system-scripts
+echo "${GREEN}Creating release branch...${NC}"
+DATE=$(date "+%Y-%m-%d")
+BRANCH="release-${DATE}"
+git checkout -b $BRANCH
 
-if git diff-index --quiet HEAD --; then
-  echo "${RED}No local changes detected, therefore version bump did not occur. Exiting...${NC}"
+echo "${GREEN}Bumping version...${NC}"
+PRE_VERSION_HASH=$(git rev-parse HEAD)
+yarn lerna version --no-push
+POST_VERSION_HASH=$(git rev-parse HEAD)
+
+if [ "$PRE_VERSION_HASH" = "$POST_VERSION_HASH" ]; then
+  echo "${RED}No bump commit detected. Removing release branch and exiting...${NC}"
+  git checkout -
+  git branch -D $BRANCH
   exit 1
 fi
 
 echo "${GREEN}Pushing tag and release commit to Github...${NC}"
-PACKAGE_VERSION=$(node -pe "require('./lerna.json').version")
-TAG_PREFIX=$(node -pe "require('./lerna.json').tagVersionPrefix")
-TAG="$TAG_PREFIX$PACKAGE_VERSION"
-BRANCH="release-$PACKAGE_VERSION"
-
-# Create and push release branch containing the updated package versions
-git checkout -b $BRANCH
-git add --all
-git commit -m "Bump package version to $PACKAGE_VERSION"
 git push --set-upstream origin $BRANCH
+git push origin --tags
```

Comment:
Comment this out when testing. It'll create local tags, which you will have to delete with `git tag -d [tags...]`. It'll also create a branch, which you'll have to move away from and then delete.

Reply:
Whoops, also comment the line above it that pushes the branch